src/views/product-tutorials-view/components/product-view-content/components/branded-callout/index.tsx

```diff
@@ -9,23 +35,40 @@ function BrandedCallout({
   product,
 }: BrandedCalloutProps) {
   return (
-    <div>
-      <h2 id={headingSlug}>{heading}</h2>
-      <pre className={s.placeholder}>
-        <code>
-          {JSON.stringify(
-            {
-              component: 'BrandedCallout',
-              heading,
-              subheading,
-              cta,
-              product,
-            },
-            null,
-            2
-          )}
-        </code>
-      </pre>
+    <div
+      className={s.root}
+      style={
+        {
+          '--gradient-start': `var(--token-color-${product}-gradient-faint-start)`,
+          '--gradient-stop': `var(--token-color-${product}-gradient-faint-stop)`,
+        } as CSSProperties
+      }
+    >
+      <div className={s.textContainer}>
+        <h2 id={headingSlug} className={s.heading}>
+          {heading}
+        </h2>
+        {subheading ? <p className={s.subheading}>{subheading}</p> : null}
+        <p className={s.cta}>
+          <StandaloneLink
+            text={cta.text}
+            href={cta.url}
```

Comment:
We probably need to run this url through a formatter of some sort, as the links are still pointing to old Learn paths. We probably don't know whether its a tutorial or collection url though, so we'd need to parse that 🤔

Reply:
Update here: have done a bunch of refactoring and now have `detect-and-reformat-learn-url` which uses `rewrite-tutorials-link` under the hood.